src/Momento.Sdk/Internal/Utils.cs

```diff
@@ -18,6 +26,42 @@ public static class Utils
     /// </summary>
     public const int DEFAULT_MAX_MESSAGE_SIZE = 5_243_000;
 
+    /// <summary>
+    /// Create a GrpcChannelOptions object from an IGrpcConfiguration object.
+    /// </summary>
+    /// <param name="grpcConfig">The IGrpcConfiguration object specifying underlying grpc options</param>
+    /// <param name="loggerFactory"></param>
+    /// <returns></returns>
+    public static GrpcChannelOptions GrpcChannelOptionsFromGrpcConfig(IGrpcConfiguration grpcConfig, ILoggerFactory loggerFactory) {
```

Comment:
It feels strange putting this in a generic utils class. Would it make sense to have an abstract GrpcManager for this?

Reply:
For sure, I'll give that a shot

Reply:
lol, @nand4011 , that's a very OO suggestion :) I kind of like this being a pure function since it has nothing to do with any Object's `this`.

this is not a blocker either way for me but I will always choose FP paradigm when I don't need access to member variables for an object.